cilium-dbg/cmd/bpf_multicast_subscribers.go

```diff
@@ -150,24 +150,24 @@ cilium-dbg bpf multicast subscriber delete 229.0.0.1 10.100.0.1
 
 // subscriberData is used to store the subscribers of a multicast group
 type subscriberData struct {
-	groupAddr   netip.Addr
-	subscribers []*maps_multicast.SubscriberV4
+	GroupAddr   netip.Addr
```

Comment:
I think we should leverage struct tags  for several reasons: 
1. groupAddr is not readily readable vs groupAddress (or the likes...) 
1. would make it clear for future selfs why these fields are exported in the first place. 
1. supported output formats

Reply:
@derailed do you want these changes? Please mark as "require changes". Otherwise I'll merge this as is.

Reply:
Hi @lmb I'll push a new change to address this comment. Added dont merge label for the time being.

Reply:
Updated to have json tags in the struct.